css/index.styl

```diff
@@ -1048,7 +1049,8 @@ loadingSize = 30px
     .auth0-lock-widget-container
       //mobile view
       @media screen and (max-width: 480px)
-        height 100%
+        height 100vh  /* Support browsers where custom properties are not supported */
```

Comment:
I'm not very familiar with `styl` files but I would guess that this could be a mixin or something to avoid duplication?

Reply:
Yes it can! Thanks for the idea. I've moved it into a mixin now.